Countries Info/ViewControllers/CountriesListViewController.swift

```diff
@@ -82,6 +88,12 @@ extension CountriesListViewController: UICollectionViewDelegate, UICollectionVie
         return cell
     }
     
+    func collectionView(_ collectionView: UICollectionView, didSelectItemAt indexPath: IndexPath) {
+        collectionView.deselectItem(at: indexPath, animated: true)
+        
```

Comment:
We can resolve this controller here, then pass the view model and then push via using navigation controller.
We can discuss this more if needed in our next sync up.